balboa-client-dispatcher/src/main/scala/com/socrata/balboa/config/DispatcherConfig.scala

```diff
@@ -0,0 +1,33 @@
+package com.socrata.balboa.config
+
+import com.socrata.balboa.config.ClientType.ClientType
+import com.socrata.balboa.metrics.config.{Configuration, Keys}
+import org.slf4j.LoggerFactory
+
+/**
+ * All inclusive Configuration file
+ */
+object DispatcherConfig {
+
+  private val Log = LoggerFactory.getLogger("DispatcherConfig")
+
+  /**
+   * @return Client Types parsed from configuration.
+   */
+  def clientTypes: Array[ClientType] = {
+    val clientStrings: Array[String] = Configuration.get().getString(Keys.DISPATCHER_CLIENT_TYPES).split(',')
```

Comment:
The main thing I just want to see is the list splitting functionality moved into a getList function on Configuration since it's very useful generic functionality.